hugo/content/045_contribute/20_documentation/10_organisation/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Organisation
+type: padding
+---
+
+## Content Organisation
+
+
+This site uses Hugo. In Hugo, [content organization](https://gohugo.io/content-management/organization/) is a core concept.
+
+{{% notice note %}}
+**Hugo Tip:** Start Hugo with `hugo server --navigateToChanged` for content edit-sessions.
```

Comment:
this is rendered using an extremely small font

Reply:
1aa4484c35a233bd70012f0002021730583dc7fa